src/server.cpp

```diff
@@ -248,6 +249,10 @@ int server_main(int argc, char** argv) {
         auto& config = ovms::Config::instance().parse(argc, argv);
         configure_logger(config.logLevel(), config.logPath());
 
+        if(config.cpuExtensionLibraryPath() != "") {
```

Comment:
should be moved to some other place, where we create OV::Core during normal flow. Just once.